src/elements/ButtonBlock/ButtonBlock.tsx

```diff
@@ -0,0 +1,30 @@
+import { ButtonBlockNode } from '@prezly/story-content-format';
+import classNames from 'classnames';
+
+import './ButtonBlockNode.scss';
+
+interface ButtonNodeProps {
+    node: ButtonBlockNode;
+}
+
+export function ButtonBlock({ node }: ButtonNodeProps) {
+    const { href, label, layout, new_tab: newTab, variant } = node;
+
+    return (
+        <a
+            className={classNames('prezly-slate-button-block', {
+                'prezly-slate-button-block--disabled': !href,
+                'prezly-slate-button-block--outlined': variant === ButtonBlockNode.Variant.OUTLINE,
+                'prezly-slate-button-block--fullWidth': layout === ButtonBlockNode.Layout.WIDE,
```

Comment:
Let's match the class name with the layout name:

```suggestion
                'prezly-slate-button-block--wide': layout === ButtonBlockNode.Layout.WIDE,
```